src/components/telemetry_monitor/src/metric_wrapper.cc

```diff
@@ -68,6 +68,10 @@ void MetricWrapper::Clear() {
   }
 }
 
+const utils::ResourseUsage& MetricWrapper::getResources() {
+  return *resources;
```

Comment:
@BSolonenko `getResources()` is a part of public interface: therefore it is potential core crash with `SEGMENATION_FAULT`

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/pull/30/commits/5e15fe123252c49dde19d2d32cd757f1ac7eacf8